src/Commands/Install.php

```diff
@@ -12,42 +12,97 @@ class Install extends Command
+                '--tag' => 'config',
             ]);
-            $this->info('✔️  Created routes/tenant.php');
+        });
+
+        $this->patienceIsKeyToLife();
+        $this->newLine();
+
+        if (!file_exists(base_path('routes/tenant.php'))) {
+            $this->components->task('Publishing routes', function () {
+                $this->callSilent('vendor:publish', [
+                    '--provider' => 'Stancl\Tenancy\TenancyServiceProvider',
+                    '--tag' => 'routes',
+                ]);
+            });
+            $this->newLine();
         } else {
-            $this->info('Found routes/tenant.php.');
+            $this->components->warn('File [routes/tenant.php] already existe.');
         }
 
-        $this->callSilent('vendor:publish', [
-            '--provider' => 'Stancl\Tenancy\TenancyServiceProvider',
-            '--tag' => 'providers',
-        ]);
-        $this->info('✔️  Created TenancyServiceProvider.php');
-
-        $this->callSilent('vendor:publish', [
-            '--provider' => 'Stancl\Tenancy\TenancyServiceProvider',
-            '--tag' => 'migrations',
-        ]);
-        $this->info('✔️  Created migrations. Remember to run [php artisan migrate]!');
-
-        if (! is_dir(database_path('migrations/tenant'))) {
-            mkdir(database_path('migrations/tenant'));
-            $this->info('✔️  Created database/migrations/tenant folder.');
+        $this->patienceIsKeyToLife();
+
+        $this->components->task('Publishing providers', function () {
+            $this->callSilent('vendor:publish', [
+                '--provider' => 'Stancl\Tenancy\TenancyServiceProvider',
+                '--tag' => 'providers',
+            ]);
+        });
+
+        $this->patienceIsKeyToLife();
+        $this->newLine();
+
+        $this->components->task('Publishing migrations', function () {
+            $this->callSilent('vendor:publish', [
+                '--provider' => 'Stancl\Tenancy\TenancyServiceProvider',
+                '--tag' => 'migrations',
+            ]);
+        });
+
+        $this->patienceIsKeyToLife();
+        $this->newLine();
+
+        if (!is_dir(database_path('migrations/tenant'))) {
+            $this->components->task('Creating database/migrations/tenant folder', function () {
+                mkdir(database_path('migrations/tenant'));
+            });
+        } else {
+            $this->components->warn('Folder [database/migrations/tenant] already existe.');
         }
 
-        $this->comment('✨️ stancl/tenancy installed successfully.');
+        $this->components->info('✨️ stancl/tenancy installed successfully.');
+
+        $this->patienceIsKeyToLife();
+        $this->askForSupport();
+
+        return 0;
+    }
+
+    /**
+     * Pause the console execution for 1 second, help the user to have time and read the output
+     *
+     * @return void
+     */
+    public function patienceIsKeyToLife(): void
+    {
+        sleep(1);
+    }
```

Comment:
Get rid of this method, we can just use `sleep(1)` inline. Also, the output isn't long, so I don't think there's a reason to add multiple pauses to the installation command?

Currently the command feels instant and the output is short and readable. So making it feel like there's a lot happening in the background by adding `sleep()`s is probably not worth it.